package.json

```diff
@@ -4,7 +4,10 @@
   "description": "this is a simple application for getting the weather based on your city built with javaScript",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "webpack --mode development",
+    "build": "webpack --mode production",
+    "start": "webpack-dev-server --mode development --open"
```

Comment:
You need to list `webpack-dev-server` as a dependency, otherwise, it will throw an error when the script is run. Either remove this script or add the package as dev dependency